bundles/org.openhab.binding.pentair/src/main/resources/OH-INF/thing/intellichem.xml

```diff
@@ -96,34 +96,54 @@
 		<item-type>Number</item-type>
 		<label>Tank Level</label>
 		<description>Tank level (1-7).</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Level</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="calciumHardness">
 		<item-type unitHint="ppm">Number:Dimensionless</item-type>
 		<label>Calcium Hardess</label>
 		<description>Calcium hardness (ppm).</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>AirQuality</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="cyaReading">
 		<item-type unitHint="ppm">Number:Dimensionless</item-type>
 		<label>CYA Reading</label>
 		<description>Cyanuric acid reading (ppm).</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>AirQuality</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="alkalinity">
 		<item-type unitHint="ppm">Number:Dimensionless</item-type>
 		<label>Total Alkalinity</label>
 		<description>Total Alkalinity reading (ppm).</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>AirQuality</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="phDoserType">
 		<item-type>String</item-type>
 		<label>PH Doser Type</label>
 		<description>The doser type for PH (None, CO2, Acid).</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>CO2</tag>
```

Comment:
This isn't an measurement, but designates what the pool system uses for pH balancing.
My guess is a Status/Mode would be better